.gitattributes

```diff
@@ -5,5 +5,4 @@
 build/** linguist-generated=false
 
 # Git large file storage
-*.psd filter=lfs diff=lfs merge=lfs -text
-**/fixtures/lfs-traces/** filter=lfs diff=lfs merge=lfs -text
+**/fixtures/lfs-traces/** filter=lfs diff=lfs merge=lfs
```

Comment:
Good call removing -text... I think? I'm guessing here, but does this change mean that git will never attempt to auto-resolve a conflict on these files, right (treat it like binary?)

Reply:
shrug... i read this:

>  finally the -text argument tells Git that this is not a text file, which is a strange way to tell it that it's a binary file.

but from a not totally reliable source. 

a nice source i did find was https://github.com/git-lfs/git-lfs/issues/4881#issuecomment-1041581286

Reply:
if they are adding it by default, seems we should keep it.